docs/content/docs/guide/running.md

```diff
@@ -114,14 +114,14 @@ entire suite of checks once again.
 
 ![github apps rerun check](/images/github-apps-rerun-checks.png)
 
-### GitOps command on pull or merge request
+## GitOps commands
 
-If you are targeting a push, pull or merge request you can use `GitOps` comment
-inside your pull request, to restart all or specific Pipelines.
+The GitOps commands are a way to trigger Pipelines-as-Code actions via comments
+on a `Pull Request` or comment on a `Push`ed commit.
 
-For example, you want to restart all your pipeline you can add a comment starting
-with `/retest` and all PipelineRun attached to that pull or merge request will be
-restarted :
+For example, you want to restart all your pipelineruns you can add a comment
+starting with `/retest` and all PipelineRuns attached to that pull or merge
+request will be restarted :
```

Comment:
How about including one more section under GitOps commands for pull or merge request
something like below

## GitOps commands

The GitOps commands are a way to trigger Pipelines-as-Code actions via comments
on a `Pull Request` or comment on a `Push`ed commit.

### GitOps commands on pull or merge request

If you want to trigger a GitOps command on a pull or merge request, you can include the
`GitOps` comments inside your pull or merge request. These comments can be used to
restart either all pipelines or specific ones.

For example, you want to restart all your pipelineruns you can add a comment
starting with `/retest` and all PipelineRuns attached to that pull or merge
request will be restarted :